modules/imgproc/src/remap_type.hpp

```diff
@@ -0,0 +1,66 @@
+namespace cv {
```

Comment:
What's the point of adding such a tiny file? Can we just put the content into imgwarp.cpp?

Reply:
`imgwarp.cpp` contains 3700 lines. This file is not very user-friendly)